app/models/user.rb

```diff
@@ -60,6 +61,10 @@ def password_reset_expired?
     reset_sent_at < Settings.users.expire_time_reset_pass.hours.ago
   end
 
+  def feed
+    Micropost.where "user_id = ?", id
```

Comment:
```suggestion
    Micropost.where "user_id = ?", id
```
đưa cái này viết vào scope trong model micropost nha em